packages/@aws-cdk/aws-pipes-targets-alpha/lib/index.ts

```diff
@@ -1 +1,2 @@
 export * from './sqs';
+export * from './step-function';
```

Comment:
Please make sure these are named consistently 
```suggestion
export * from './stepfunctions';
```

Reply:
Thanks for taking a look already :)

Currently its consistent using `step-function` everywhere. Should I change it to the plural version of it?

Reply:
How do you mean "everywhere"? I was referring to the module name which is `aws-stepfunctions`.

(You might totally be right about this by the way)

Reply:
>How do you mean "everywhere"?

I meant the filenames. 

> I was referring to the module name which is aws-stepfunctions.

Ah I think I know now what you meant. The package name is `aws-stepfunctions` and I should keep that name consistent also in this pipes-targets module, changed it to stepfunctions :)

I kept the name `StepFunctionTarget` if you think I should change that as well let me know.